chatbot.py

```diff
@@ -21,6 +21,12 @@ def setup():
     return clf, learn_response
 
 def message_to_bot(H,clf,learn_response):
+    if learner_response==1:
```

Comment:
Is this supposed to be `learner_response==1` or `learn_response==1`?

Reply:
learner_respone